crates/core/src/db/relational_db.rs

```diff
@@ -143,10 +145,10 @@ impl RelationalDB {
             commit_log,
             _lock: Arc::new(lock),
             address: db_address,
-            row_count_fn: Arc::new(move |table| {
+            row_count_fn: Arc::new(move |table_id, table_name| {
```

Comment:
Similarly `table_id` can be removed from this lambda and the below call sites once this change has been released.